python/generator_utils.py

```diff
@@ -86,6 +86,40 @@ def __str__(self):
 
 class MemberVariable:
   """Simple class to hold information about a member variable"""
+
+  def parse_julia_type(self, **kwargs):
+    """Parse the given c++ type to a Julia type"""
+    self.cpp_type = kwargs.pop('cpp_type', None)
+    is_array = kwargs.pop('is_array', False)
+
+    builtin_types_map = {"int": "Int32", "float": "Float32", "double": "Float64",
+                         "bool": "Bool", "long": "Int32", "unsigned int": "UInt32",
+                         "unsigned long": "UInt32", "char": "Char", "short": "Int16",
+                         "long long": "Int64", "unsigned long long": "UInt64", "std::string": "String"}
```

Comment:
```suggestion
                         "long long": "Int64", "unsigned long long": "UInt64"}
```
Since `std::string` has been marked deprecated in #305 I don't think we need to support that in the Julia interface. This might also make some of the handling below a bit easier to do.